config/locales/zh.yml

```diff
@@ -1224,10 +1224,13 @@ in_person_proofing.body.cta.button: 尝试亲身去
 in_person_proofing.body.cta.prompt_detail: 无法在网上完成这一步骤的大多数人都能在一个参与本项目的邮局成功地证实身份。去邮局无需预约。全国各地都有参与本项目的邮局。
 in_person_proofing.body.expect.heading: 去邮局后会发生什么
 in_person_proofing.body.expect.info: 你去邮局后 24 小时内，我们会给你发电邮，告诉你是否成功验证了身份。
+in_person_proofing.body.location.change_location_heading: 需要更改你的邮局地点吗？
+in_person_proofing.body.location.change_location_info: 你不需要创建新的条形码，你可以将这个条形码带到任何参与本项目的邮局。 %{find_other_locations_link}
+in_person_proofing.body.location.change_location_find_other_locations: 查找其他参与本项目的邮局地点
```

Comment:
Nit: We're missing a "。" at the end of the sentence. So the end of the sentence should look like:

> ... 点。